ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarDemoController_SwiftUI.swift

```diff
@@ -0,0 +1,184 @@
+    @State var useAlternateBackground: Bool = false
+    @State var isOutOfOffice: Bool = false
+    @State var isRingVisible: Bool = true
+    @State var isTransparent: Bool = true
+    @State var hasPointerInteraction: Bool = false
+    @State var hasRingInnerGap: Bool = true
+    @State var primaryText: String = "Kat Larrson"
+    @State var secondaryText: String = ""
+    @State var presence: MSFAvatarPresence = .none
+    @State var showImage: Bool = false
+    @State var showImageBasedRingColor: Bool = false
+    @State var size: MSFAvatarSize = .xxlarge
+    @State var style: MSFAvatarStyle = .default
+
+    let switchToggleStyle: SwitchToggleStyle = {
+        if #available(iOS 14.0, *) {
+            // Workaround for SwiftUI Toggles that switch to green tint color
+            // when they're toggled for the first time
+            return SwitchToggleStyle(tint: Color(Colors.communicationBlue))
+        } else {
+            return SwitchToggleStyle()
+        }
+    }()
+
+    public var body: some View {
+        VStack {
+            AvatarView(style: .default,
+                       size: .xxlarge,
+                       image: nil,
+                       primaryText: primaryText,
+                       secondaryText: secondaryText)
+                .size(size)
+                .style(style)
+                .image(showImage ? UIImage(named: "avatar_kat_larsson") : nil)
+                .isRingVisible(isRingVisible)
+                .hasRingInnerGap(hasRingInnerGap)
+                .imageBasedRingColor(showImageBasedRingColor ? AvatarDemoController.colorfulCustomImage : nil)
+                .isTransparent(isTransparent)
+                .presence(presence)
+                .isOutOfOffice(isOutOfOffice)
+                .hasPointerInteraction(hasPointerInteraction)
+                .frame(maxWidth: .infinity, minHeight: 150, alignment: .center)
+                .background(useAlternateBackground ? Color.gray : Color.clear)
+
+            ScrollView {
+                Group {
+                    Group {
+                        VStack(spacing: 0) {
+                            Text("Content")
+                                .frame(maxWidth: .infinity, alignment: .leading)
+                                .font(.title)
+                            Divider()
+                        }
+
+                        TextField("Primary Text", text: $primaryText)
+                            .autocapitalization(.none)
+                            .disableAutocorrection(true)
+                            .textFieldStyle(RoundedBorderTextFieldStyle())
+
+                        TextField("Secondary Text", text: $secondaryText)
+                            .autocapitalization(.none)
+                            .disableAutocorrection(true)
+                            .textFieldStyle(RoundedBorderTextFieldStyle())
+
+                        Toggle("Set image", isOn: $showImage)
+                            .toggleStyle(switchToggleStyle)
```

Comment:
Nit: just to make this a bit cleaner (and while we don't yet have a vNext toggle control), would it be easier to define a custom view that automatically applies this property?